.zenodo.json

```diff
@@ -87,6 +105,21 @@
         {
             "name": "Roy Smart"
         },
+        {
+            "name": "Mihail Bankov"
+        },
+        {
+            "name": "P. L. Lim",
+            "affiliation": "Space Telescope Science Institute, USA"
+        },
+        {
+            "name": "Samuel J. Van Kooten",
+            "orcid": "0000-0002-4472-8517",
+            "affiliation": "Southwest Research Institute"
+        },
+        {
+            "name": "Sanvi Sharma"
```

Comment:
@ciaokitty Do you have an affiliation and orcid you would like us to include here?